gitlab-pipeline/stage/release.yml

```diff
@@ -178,6 +178,44 @@ release_binary_tools:automatic:
       - $PUBLISH_RELEASE_AUTOMATIC == "true"
   extends: .template_release_binary_tools
 
+.template_release_mender-monitor:
+  stage: release
+  image: debian:buster
+  variables:
+    S3_BUCKET_NAME: "mender-binaries"
+    S3_BUCKET_PATH: "mender-monitor/yocto"
+  dependencies:
+    - init:workspace
+    - init:mender-monitor-package
+  before_script:
+    # Install dependencies
+    - apt update && apt install -yyq awscli git wget python3 python3-pip
+    - wget https://raw.githubusercontent.com/mendersoftware/integration/master/extra/requirements.txt
+    - pip3 install -r requirements.txt
+    # Restore workspace from init stage
+    - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
+    - mv workspace.tar.gz stage-artifacts /tmp
```

Comment:
Question:
If they're not in the workspace, why move them? :thinking:

Reply:
I don't understand your question. this is what we do in absolutely all jobs in the pipeline. [One example](https://github.com/mendersoftware/mender-qa/blob/9666919d9b24b68d276d074cbca18dd9b01b0ad0/gitlab-pipeline/shared/build_and_test_acceptance.yml#L31-L35).

Reply:
I'm missing something, but it doesn't matter